op-batcher/batcher/channel_manager_test.go

```diff
@@ -122,7 +122,7 @@ func ChannelManager_Clear(t *testing.T, batchType uint) {
 	// clearing confirmed transactions, and resetting the pendingChannels map
 	cfg.ChannelTimeout = 10
 	cfg.InitRatioCompressor(1, derive.Zlib)
-	m := NewChannelManager(log, metrics.NoopMetrics, cfg, defaultTestRollupConfig)
+	m := NewChannelManager(log, &metrics.TestMetrics{}, cfg, defaultTestRollupConfig)
```

Comment:
Wondering if we want to use the real metrics instead of the test metrics? Otherwise this test only asserts that the test metrics do the clearing correctly, but the real metrics stay uncovered by tests.

Reply:
[863a883](https://github.com/ethereum-optimism/optimism/pull/14780/commits/863a883f6282eff465db808d0968113623ef9857) good call